src/Faithlife.Tracing.AspNetWebApi/Faithlife.Tracing.AspNetWebApi.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <VersionPrefix>0.1.0</VersionPrefix>
+    <TargetFrameworks>net461</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\Faithlife.Tracing.AspNetMvc\AspNetTracing.cs" />
+    <Compile Include="..\Faithlife.Tracing.AspNetMvc\RequestActionTraceProvider.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="System.Web" />
+    <PackageReference Include="Microsoft.AspNet.WebApi.Core" Version="4.0.30506" />
+    <PackageReference Include="Microsoft.AspNet.WebApi.WebHost" Version="4.0.30506" />
```

Comment:
Pretty old version. Is 5.x.x a breaking change? If so, I wonder if it would be better to call this assembly `AspNetWebApi4` so that projects can choose a separate `AspNetWebApi5` in the future.

Reply:
I just searched across `packages.config` (in GitHub) and most of our projects use 5.2.3, with a few on 5.1.2, and a handful on 4.0.30506. The breaking changes (if any) don't appear to be in the action filter pipeline, so it would probably be best to update this to 5.2.3 and clients who can't update to 5.x can binding-redirect back to 4.0.30506?